helm/hpcc/templates/_warnings.tpl

```diff
@@ -0,0 +1,59 @@
+{{/*
+
+---  DO NOT EDIT THIS FILE ---
+
+##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+##############################################################################
+
+*/}}
+{{/*
+Generate a list of warnings
+*/}}
+{{- define "hpcc.getWarnings" -}}
+  {{- $warnings := list -}}
+  {{- $root := . -}}
+  {{- $defaultImage :=  include "hpcc.imageName" (dict "root" $ "me" dict ) | quote -}}
+  {{- $defaultVersion := .Values.global.image.version | default .Chart.Version -}}
+  {{ if (ne $defaultVersion .Chart.Version) -}}
+    {{ $warning := dict "source" "helm" "severity" "warning" -}}
```

Comment:
trivial: might not make a material difference, but as this helper func. is not outputting anything, should consistently chomp leading and trailing whitespace on ech template line, i.e. above 2 lines should have {-